ArticleTemplates/assets/js/modules/experiments/ab.js

```diff
@@ -1,34 +1,23 @@
-define([
-], function (
-) {
-    'use strict';
+//  --- EXAMPLE ---
+// var tests = {
+//     lowFrictionParticipation: lowFrictionParticipation.init
+// }
 
-    /* --- EXAMPLE START --- */
+const tests = {};
 
-    // var tests = { 
-    //     lowFrictionParticipation: lowFrictionParticipation.init
-    // }
+function init() {
+    let key;
+    let testSpec;
```

Comment:
I've changed a lot of `var` to `let` but I think it's a good suggestion to go back over and update the variables that could be `const` and the positioning of them. I might save that for a later PR so that they're all consistent!